### PR TITLE
HOSS2-387-Update error messages for names and NINO

### PR DESCRIPTION
**Pull request by AdamCoops**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -46,11 +46,12 @@ error.dateOfBirth.year.zero='Year must not be a 0'
...
-error.nino.invalid-format='Invalid National Insurance number'
-error.familyName.required='Enter surname'
-error.familyName.invalid-format='Invalid surname'
+error.nino.invalid-format='National Insurance number must only include letters and numbers'
+error.familyName.required='Enter a last name'
+error.familyName.invalid-format='Last name must only include letters, hyphens and numbers 0 to 9'
+error.familyName.length='Last name must be 2 or more characters'
 error.givenName.required='Enter first name'
-error.givenName.invalid-format='Invalid first name'
+error.givenName.invalid-format='First name must only include letters, hyphens and numbers 0 to 9'
```
Comment (liampan) [inline conf/messages:54]: is this content correct? Names cant contain numbers, Alpha or ' - or space

**Review by liampan — COMMENTED** [review]
functionally all looks good, just worth questioning that content as it disagrees with the validation

File: conf/messages (hunk 1 above)
Comment (AdamCoops) [reply conf/messages:54]: "First name must only include letters, hyphens and numbers 0 to 9" is what's written on the ticket.

Comment (AdamCoops) [reply conf/messages:54]: Liam's approved this. To be reviewed by the content folks in the new year.

**Review by liampan — APPROVED** [review]
Approved pending content change, however please merge when tested to allow for NFT

**Pull request merged**